src/detect-geoip.c

```diff
@@ -52,6 +52,8 @@ static int DetectGeoipSetupNoSupport (DetectEngineCtx *a, Signature *b, const ch
 void DetectGeoipRegister(void)
 {
     sigmatch_table[DETECT_GEOIP].name = "geoip";
+    sigmatch_table[DETECT_GEOIP].desc = "enables (you) to match on the source, destination or source and destination IP addresses of network traffic, and to see to which country it belongs.";
```

Comment:
the 'enables (you)' part is a bit of a different style. What about just 'match on the source...'?